assets/queries/cloudFormation/amplify_app_access_token_rule/test/negative.yaml

```diff
@@ -0,0 +1,59 @@
+Resources:
+     NewAmpApp-2:
```

Comment:
Remove `-`
Linter message: `Resource NewAmpApp-2 has an invalid name.  Name has to be alphanumeric.`